projects/kit/components/slider/slider.style.less

```diff
@@ -1,12 +1,129 @@
-.dot:first-child {
-    display: none;
+/*
+*** Thumb – this is an element on the track that the user can move around to select varying range values.
+*** Track – this is the part of the slider that the thumb runs along.
+*** Ticks - tick marks along the length of the track.
+*/
+
+@thumb-heights: {
+    @s: 0.5rem;
+    @m: 1rem;
+};
+@track-height: 0.125rem;
+@ticks-thickness: 0.25rem;
+
+@track-color: var(--tui-base-03);
+@thumb-color: currentColor;
+
+.ticks() {
+    background: @track-color;
+    background: linear-gradient(to right, var(--tui-base-07) @ticks-thickness, @track-color @ticks-thickness) repeat-x;
+    background-size: var(--tui-slider-step-percentage) @track-height;
+    background-position: 0 center;
+}
+
+.customize-track() {
+    height: @track-height;
+    border-radius: var(--tui-radius-m);
+}
+
+.customize-thumb(@size) {
+    appearance: none;
+    background-color: @thumb-color;
+    border: none;
+    border-radius: 50%;
+    height: @thumb-heights[ @@size];
+    width: @thumb-heights[ @@size];
+
+    :not(:disabled)& {
+        cursor: ew-resize;
+    }
+
+    :focus& {
```

Comment:
Let's use `:focus-visible`. It recently shipped in Safari as well and it can be considered graceful degradation if it doesn't work.

Reply:
done